portfolio/src/main/webapp/script.js

```diff
@@ -26,3 +26,16 @@ function addRandomGreeting() {
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = greeting;
 }
+
+/**
+ * Goes back to the previous page.
+ */
+function goBack() {
+    //if it was the first page
```

Comment:
Maybe replace "was" with "is"? Keep simple present tense unless it's clearly past tense.
And need a space and capital letter. So maybe:
// If this is the first page.

Reply:
Fixed